doc/examples/planning_adapters/planning_adapters_tutorial.rst

```diff
@@ -201,7 +201,7 @@ This section has insights as to when to use which planner and how using certain
 For more information on each of these motion planners, refer to their individual tutorial pages
 :doc:`OMPL </doc/examples/ompl_interface/ompl_interface_tutorial>`,
 :doc:`CHOMP </doc/examples/chomp_planner/chomp_planner_tutorial>` and
-:doc:`STOMP </doc/examples/stomp_planner/stomp_planner_tutorial>`.
+.. :doc:`STOMP </doc/examples/stomp_planner/stomp_planner_tutorial>`.
```

Comment:
Looks like this change might be the reason its failing CI.

Reply:
I tried commenting out that link there... that tutorial doesn't exist on main.